apps/dashboard/src/main/java/com/akto/action/AdminSettingsAction.java

```diff
@@ -243,6 +243,23 @@ public String editPrivateCidrList(){
             AccountSettingsDao.instance.getMCollection().updateOne(
                 AccountSettingsDao.generateFilter(), Updates.set(AccountSettings.PRIVATE_CIDR_LIST, privateCidrList)
             );
+
+            int accountId = Context.accountId.get();
```

Comment:
This job will run everytime the user presses enter. Fix: add an apply button in UI instead of making an API call everytime.